package.json

```diff
@@ -18,15 +18,15 @@
   },
   "scripts": {
     "test": "mocha build/test --recursive",
+    "build": "npm run clean && npm run compile && npm run docs",
     "conformance": "./run_conformance_tests.sh",
     "check": "gts check",
-    "clean": "gts clean",
-    "compile": "rm -rf ./build && tsc -p .",
+    "clean": "rm -rf ./build",
```

Comment:
There shouldn't be a need to replace `gts` with a manual rimraf of the build folder declared in our tsconfig.

`gts clean` is the expected clean command we use in client libs and elsewhere in this library. It also provides better output via gts.

- https://github.com/google/gts/blob/main/src/clean.ts
- https://github.com/GoogleCloudPlatform/functions-framework-nodejs/blob/master/tsconfig.json#L5

This change is also a little tangential to the api-extractor change.

Reply:
I changed it because `gts clean` doesn't work on my machine:

```
❯ npm run clean

> @google-cloud/functions-framework@1.10.1 clean /usr/local/google/home/mattrobertson/gcf/functions-framework-nodejs
> gts clean

version: 16
Error: Error: /usr/local/google/home/mattrobertson/gcf/functions-framework-nodejs/tsconfig.json
Error: /usr/local/google/home/mattrobertson/gcf/functions-framework-nodejs/gts/tsconfig-google.json
ENOENT: no such file or directory, open '/usr/local/google/home/mattrobertson/gcf/functions-framework-nodejs/gts/tsconfig-google.json'
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/functions-framework@1.10.1 clean: `gts clean`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @google-cloud/functions-framework@1.10.1 clean script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /usr/local/google/home/mattrobertson/.npm/_logs/2021-11-10T17_43_50_564Z-debug.log
```

@grant does it work for you right now?

Reply:
looks like our tsconfig should actually be extending `./node_modules/gts/tsconfig-google.json`:

https://github.com/googleapis/nodejs-pubsub/blob/main/tsconfig.json#L2

I wonder how `tsc` is even working now...

Reply:
OK. It looks like that's a different issue.

It looks like this expects a global `gts` or something maybe changed when this was first created with linking. We can change it to the local gts with `tsconfig`:

```
"extends": "./node_modules/gts/tsconfig-google.json",
```

This config looks the same as other client libs.

We probably shouldn't inline rm rf because one of our scripts doesn't work as expected.